content/en/partners/cloud_cost_management/aws.md

```diff
@@ -0,0 +1,193 @@
+    tag: "Blog"
+    text: "Understand your Kubernetes and ECS spend with Datadog Cloud Cost Management"
+  - link: "https://www.datadoghq.com/blog/google-cloud-cost-management/"
+    tag: "Blog"
+    text: "Empower engineers to take ownership of Google Cloud costs with Datadog"
+  - link: "https://www.datadoghq.com/blog/total-cost-of-service-ownership-ccm/"
+    tag: "Blog"
+    text: "Quickly and comprehensively analyze the cloud and SaaS costs behind your services"
+    text: "Learn about Tags in Cloud Cost Management"
+  - link: "https://www.datadoghq.com/blog/cloud-costs-study-learnings/"
+    tag: "Blog"
+    text: "Key learnings from the State of Cloud Costs study"
+  - link: "https://www.datadoghq.com/blog/unit-economics-ccm/"
+    tag: "Blog"
+    text: "Monitor unit economics with Datadog Cloud Cost Management"
+  - link: "https://www.datadoghq.com/blog/finops-at-datadog/"
+    tag: "Blog"
+    text: "How we've created a successful FinOps practice at Datadog"
+  - link: "https://www.datadoghq.com/blog/cloud-cost-management-saved-millions/"
+    tag: "Blog"
+    text: "How we saved $1.5 million per year with Cloud Cost Management"
+  - link: "https://www.datadoghq.com/blog/cloud-cost-management-oci/"
+    tag: "Blog"
+    text: "Manage and optimize your OCI costs with Datadog Cloud Cost Management"
+cascade:
+    algolia:
+      subcategory: 'Cloud Cost Management'
+      tags: ['cloud cost', 'cloud integrations', 'cloud cost management', 'cloud cost aws']
+
+---
+## Overview
+
+This guide explains how AWS Managed Service Providers (MSPs) configure AWS Billing Conductor and generate Pro Forma Cost and Usage Reports (CURs) so that a customer’s Member Account can be connected to Datadog CCM without exposing the MSP’s Management Account.
+
+The steps in this document describe what **the MSP partner** configures and what **the customer (Member Account owner)** must configure.
+
+For general (non-MSP) CCM setup instructions, see the [Datadog CCM AWS setup guide][1].
+
+## Billing conductor
+[AWS Billing Conductor][11] is a custom billing service for AWS Marketplace Channel Partners and organizations that have chargeback requirements.
+Billing Conductor enables customers to create a second, pro forma version of their costs to share with their customers or account owners.
+Billing rates, credits and fees, and overhead costs can be customized at your discretion. You can also select which accounts to include in the CUR.
+
+**Important limitations**: 
+- Pro forma Cost and Usage Reports do not include discounts and taxes, which makes it difficult to compare costs in Datadog to AWS Cost Explorer.
+- Adding accounts to a billing group impacts how Reservations and Savings Plans are shared across AWS accounts.
+
+To create a billing conductor CUR, follow the [AWS Cost and Usage Reports user guide][12]. Ensure the CUR meets [Datadog's requirements][2].  
+
+## Prerequisites
+
+As an MSP partner, verify the following before starting:
```

Comment:
it feels like the headers here are too big for a short checklist. there also needs to be more parallelism between the section headers. i would try something like this:

**MSP access**: Access to the Management Account to configure Billing Conductor (Pricing Rules, Pricing Plans, Billing Groups).

**Customer permissions**: The customer must have permission to:
- deploy CloudFormation stacks
- create and manage Cost and Usage Reports (CURs)
- grant Datadog access to the S3 bucket containing the CUR

**Datadog integration**: The AWS integration must be installed for the Member Account in Datadog. For more information, see the [AWS integration setup instructions][5].